app/components/menubar/newMenuBar.js

```diff
@@ -156,7 +158,16 @@ const MobileNav = ({ nav_Items, nft }) => {
           </Offcanvas.Body>
         </Navbar.Offcanvas>
         <Navbar.Brand className={styles.brand}>
-          {nft ? <NFTProfilePicture id='img2' /> : <DummyLoginButton />}
+          {nft ? <NFTProfilePicture id='img2' /> :
```

Comment:
Nice catch! You have added the RC Auth here.

Reply:
Haha yes! I needed that for showing how we can use auth data for for embedded chat component